kvm-ioctls/src/ioctls/vcpu.rs

```diff
@@ -876,8 +876,8 @@ impl VcpuFd {
     /// This function is unsafe because there is no guarantee `xsave` is allocated with enough space
     /// to hold the entire xsave state.
     ///
-    /// The required size can be retrieved via `KVM_CHECK_EXTENSION(KVM_CAP_XSAVE2)` and can vary
-    /// depending on features that have been dynamically enabled by `arch_prctl()`. Thus, any
+    /// The required size in byte can be retrieved via `KVM_CHECK_EXTENSION(KVM_CAP_XSAVE2)` and can
```

Comment:
```suggestion
    /// The required size in bytes can be retrieved via `KVM_CHECK_EXTENSION(KVM_CAP_XSAVE2)` and can
```

Reply:
oh well, I thought the singular form is right when using as a unit XD
fixed!